fvm/blueprints/contractDeployment.go

```diff
@@ -0,0 +1,38 @@
+package blueprints
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+const ContractDeploymentAuthorizedAddressesPathDomain = "storage"
+const ContractDeploymentAuthorizedAddressesPathIdentifier = "authorizedAddressesToDeployContracts"
+
+const setContractDeploymentAuthorizersTransactionTemplate = `
+transaction {
+		prepare(signer: AuthAccount) {
+			// this drops previous list
+			signer.load<[Address]>(from: /%s/%s)
+			signer.save([%s], to: /%s/%s)
+		}
+}
+`
+
+// SetContractDeploymentAuthorizersTransaction returns a transaction for setting storage path ...
+func SetContractDeploymentAuthorizersTransaction(serviceAccount flow.Address, authorized []flow.Address) *flow.TransactionBody {
+	addStrs := make([]string, 0)
+	for _, a := range authorized {
+		addStrs = append(addStrs, a.HexWithPrefix()+" as Address")
+	}
+
+	return flow.NewTransactionBody().
+		SetScript([]byte(fmt.Sprintf(setContractDeploymentAuthorizersTransactionTemplate,
+			ContractDeploymentAuthorizedAddressesPathDomain,
+			ContractDeploymentAuthorizedAddressesPathIdentifier,
+			strings.Join(addStrs, ", "),
+			ContractDeploymentAuthorizedAddressesPathDomain,
+			ContractDeploymentAuthorizedAddressesPathIdentifier))).
```

Comment:
Same as above, use transaction arguments instead of string formatting

Reply:
fixed at 9cb4197